templates/timesheet/index.html.twig

```diff
@@ -66,6 +66,15 @@
             'project': 'hidden-xs hidden-sm',
             'activity': 'hidden-xs hidden-sm',
             'description': 'hidden-xs hidden-sm',
+        }) %}
+
+        {% if use_tags %}
```

Comment:
Yes, because using tags should be possible for every user. By using the REST-API it's a little bit different. If you have the opinion it's not necessary I would remove the permission.